app.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express')
+const app = express()
+const { getTopics } = require('./controllers/topics.controller')
+
+
+
+app.get('/api/topics', getTopics)
+
+
+
+app.all('/api/*', (req, res, next) => {
+    res.status(404).send('Not Found')
+})
+
```

Comment:
We don't need this app.use to pass any tests right now, so it shouldn't really be here. It might not be a bad idea to replace this with the 500 catch all error handling middleware and console log the error instead.